units/crons.go

```diff
@@ -78,5 +79,32 @@ func StartCrons(ctx context.Context, env cedar.Environment) error {
 		return queue.Put(job)
 	})
 
+	if rpcTLS {
+		amboy.IntervalQueueOperation(ctx, remote, 24*time.Hour, time.Now(), opts, func(queue amboy.Queue) error {
+			return queue.Put(NewServerCertRotationJob())
```

Comment:
we should have the starting time (the thing that's now in the args) be a fixed time (e.g. midnight) to avoid having this happen every time we restart the server(? right?) 

also I think we can use the close method on the environment to cancel the master context here too.

Reply:
How about doing time.Now() + 12 hours?